src/network/networkd-manager.c

```diff
@@ -41,6 +41,13 @@
 /* use 8 MB for receive socket kernel queue. */
 #define RCVBUF_SIZE    (8*1024*1024)
 
+static int log_message_warning_errno(sd_netlink_message *m, int err, const char *msg) {
+        const char *err_msg = NULL;
+
+        (void) sd_netlink_message_read_string(m, NLMSGERR_ATTR_MSG, &err_msg);
+        return log_warning_errno(err, "%s: %s%s%m", msg, strempty(err_msg), err_msg ? " " : "");
```

Comment:
My only doubt is how the `err_msg` strings look: do they have punctuation at the end? If not, that and %m will run together. But this is a minor detail we can adjust later if needed.

Reply:
> do they have punctuation at the end?

I guess so. At least, the error strings I checked have a period.